src/PokemonGymOwner.java

```diff
@@ -0,0 +1,14 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer {
```

Comment:
Dit is maar een kleine overerving, maar ook dit helpt om minder herhaling in je code te schrijven